client/source/DelphiLint.IDEContext.pas

```diff
@@ -568,6 +610,11 @@ function TToolsApiEditView.GetLineTracker: IIDEEditLineTracker;
   Result := TToolsApiEditLineTracker.Create(FRaw.Buffer.GetEditLineTracker);
 end;
 
+function TToolsApiEditView.GetRow: Integer;
```

Comment:
Seems like we're missing some separator comments in this file.

Reply:
The convention in this file is that there are only separator comments between methods of different types. This is because it's almost entirely composed of thin wrappers around the API, so using separator comments everywhere would contribute to the visual noise rather than lessening it.